### PR TITLE
Add MinMax to console and doc additions

### PR DESCRIPTION
**Pull request by cijothomas**
Console Exporter to show Min, Max
Doc addition to show Resource in Metric

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ LogRecordExporter to print full exception details instead of just Message, when
 using `ILogger` to log exception.
 ([#3784](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3784))
 
+Added support to print Histogram Min, Max in MetricExporter.
```
Comment (TimothyMothra) [inline src/OpenTelemetry.Exporter.Console/CHANGELOG.md:13]: nit: add link to PR

File: src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs [hunk 2]
```diff
@@ -97,7 +97,14 @@ public override ExportResult Export(in Batch<Metric> batch)
                         var bucketsBuilder = new StringBuilder();
                         var sum = metricPoint.GetHistogramSum();
                         var count = metricPoint.GetHistogramCount();
-                        bucketsBuilder.Append($"Sum: {sum} Count: {count} \n");
+                        if (metricPoint.HasMinMax())
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:100]: nit: Could be a simplified a little:

```csharp
bucketsBuilder.Append($"Sum: {sum} Count: {count} ");
if (metricPoint.HasMinMax())
{
    bucketsBuilder.Append($"Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} ");
}
bucketsBuilder.Append("\n");
```

File: docs/metrics/customizing-the-sdk/README.md [hunk 3]
```diff
@@ -427,4 +427,37 @@ Refer to the individual exporter docs to learn how to use them:
...
+is built, by calling the `Build()` method on the `MeterProviderBuilder`.
+`ResourceBuilder` offers various methods to construct resource comprising of
+multiple attributes from various sources.
+
+The snippet below shows configuring the `Resource` associated with the provider.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Metrics;
```
Comment (reyang) [inline docs/metrics/customizing-the-sdk/README.md:450]:
```suggestion
using OpenTelemetry.Metrics;
using OpenTelemetry.Resources;
```

**Review by reyang — APPROVED** [review]
LGTM assuming the minor comments are addressed.

**Review by mic-max — APPROVED** [review]

**Pull request merged**